app/controllers/parties_controller.rb

```diff
@@ -2,6 +2,20 @@ class PartiesController < ApplicationController
   def new; end
 
   def create
+    party = Party.create(party_params)
+    if params[:friends].count > 0
+      params[:friends].each do |friend|
+        party.user_parties.create(host_id: current_user.id, invitee_id: friend.to_i)
+      end
+    else
+      party.user_parties.create(host_id: current_user.id, invitee_id: 0)
+    end
```

Comment:
Will be a good idea to refactor the if/else statement into helper controller method.